docs/INSTALLATION.rst

```diff
@@ -28,11 +28,12 @@ From sources
 
 The sources for aicsimageio can be downloaded from the `Github repo`_.
 
-You can either clone the public repository:
+You can either clone the public repository and submodules:
 
 .. code-block:: console
 
     $ git clone git://github.com/AllenCellModeling/aicsimageio
+    $ git submodule update --init --recursive
 
 Or download the `tarball`_:
```

Comment:
Does anyone download the tarball?  Does the tarball include the submodule stuff?

Reply:
I have no idea but this has just been a part of the cookiecutter since _forever_.

Reply:
@JacksonMaxfield thoughts on this? Not sure how the tarball gets generated.

Reply:
GitHub generates the tarball for us.